.github/workflows/tests.yml

```diff
@@ -379,6 +379,74 @@ jobs:
       with:
         parallel-finished: true
 
+  docs:
+    needs: build-package
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ["3.9"]
```

Comment:
Use 3.11 here to match the readthedocs config?

Reply:
Same here.